plugins/outputs/cloudwatch/cloudwatch_test.go

```diff
@@ -528,57 +535,43 @@ func TestBackoffRetries(t *testing.T) {
 	c := &CloudWatch{}
 	sleeps := []time.Duration{time.Millisecond * 200, time.Millisecond * 400, time.Millisecond * 800,
 		time.Millisecond * 1600, time.Millisecond * 3200, time.Millisecond * 6400}
+	assert := assert.New(t)
 	for i := 0; i <= defaultRetryCount; i++ {
-		now := time.Now()
+		start := time.Now()
 		c.backoffSleep()
-		assert.True(t, math.Abs((time.Now().Sub(now)-sleeps[i]).Seconds()) < 1)
+		// Expect time since start is between sleeps[i]/2 and sleeps[i].
+		assert.Less(sleeps[i] / 2, time.Since(start))
+		assert.Greater(sleeps[i], time.Since(start))
```

Comment:
Well this is failing on github/macos:
```
 --- FAIL: TestBackoffRetries (49.26s)
    cloudwatch_test.go:544: 
        	Error Trace:	cloudwatch_test.go:544
        	Error:      	"200ms" is not greater than "251.13787ms"
        	Test:       	TestBackoffRetries
    cloudwatch_test.go:544: 
        	Error Trace:	cloudwatch_test.go:544
        	Error:      	"400ms" is not greater than "425.195568ms"
        	Test:       	TestBackoffRetries
FAIL
```
It consistently passes on my macos laptop:
```
go test -v -count=1 ./plugins/outputs/cloudwatch --failfast -run TestBackoffRetries               
=== RUN   TestBackoffRetries
2022/07/07 09:13:25 W! cloudwatch: 0 retries, going to sleep 153 ms before retrying.
2022/07/07 09:13:25 W! cloudwatch: 1 retries, going to sleep 250 ms before retrying.
2022/07/07 09:13:25 W! cloudwatch: 2 retries, going to sleep 563 ms before retrying.
2022/07/07 09:13:26 W! cloudwatch: 3 retries, going to sleep 885 ms before retrying.
2022/07/07 09:13:27 W! cloudwatch: 4 retries, going to sleep 1780 ms before retrying.
2022/07/07 09:13:29 W! cloudwatch: 5 retries, going to sleep 3392 ms before retrying.
2022/07/07 09:13:32 W! cloudwatch: 6 retries, going to sleep 53950 ms before retrying.
2022/07/07 09:14:26 W! cloudwatch: 0 retries, going to sleep 182 ms before retrying.
--- PASS: TestBackoffRetries (61.16s)
PASS
ok  	github.com/aws/amazon-cloudwatch-agent/plugins/outputs/cloudwatch	61.988s
```